Microsoft.Toolkit.Uwp.UI.Controls/Graph/PeoplePicker/PeoplePicker.xaml

```diff
@@ -0,0 +1,95 @@
+                    xmlns:interactivity="using:Microsoft.Xaml.Interactivity">
+
+    <Style TargetType="controls:PeoplePicker">
+        <Setter Property="Background" Value="{ThemeResource ApplicationPageBackgroundThemeBrush}" />
+        <Setter Property="Foreground" Value="{ThemeResource ApplicationForegroundThemeBrush}" />
+        <Setter Property="FontFamily" Value="{ThemeResource ContentControlThemeFontFamily}" />
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="controls:PeoplePicker">
+                    <StackPanel Name="PeoplePickerContainer" DataContext="{Binding RelativeSource={RelativeSource Mode=TemplatedParent}}">
+                        <Grid>
+                            <TextBox Name="SearchBox"
+                                     Margin="0"
+                                     AutomationProperties.Name="Keywords"
+                                     PlaceholderText="{TemplateBinding PlaceholderText}" />
+                            <ProgressRing Name="Loading"
+                                          Margin="0,0,30,0"
+                                          HorizontalAlignment="Right"
+                                          AutomationProperties.Name="Progress"
+                                          FlowDirection="LeftToRight"
+                                          IsActive="False" />
+                        </Grid>
+                        <Grid>
+                            <StackPanel>
+                                <ListBox Name="SelectionsListBox"
+                                         AutomationProperties.Name="Selections"
+                                         ItemsSource="{TemplateBinding Selections}">
+                                    <ListBox.ItemTemplate>
+                                        <DataTemplate>
+                                            <Grid AutomationProperties.Name="{Binding DisplayName}">
+                                                <Grid.ColumnDefinitions>
+                                                    <ColumnDefinition Width="Auto" />
+                                                    <ColumnDefinition Width="*" />
+                                                    <ColumnDefinition Width="Auto" />
+                                                </Grid.ColumnDefinitions>
+                                                <controls:ProfileCard Grid.Row="0"
+                                                                      Grid.Column="0"
+                                                                      DisplayMode="LargeProfilePhotoLeft"
+                                                                      UserId="{Binding Id}" />
+                                                <TextBlock Grid.Row="0"
+                                                           Grid.Column="1" />
+                                                <Button Grid.Row="0"
```

Comment:
Maybe just give this button a name and grab it as a template part to register a click handler?  Then we can remove the DelegateCommand for now, since this is the only usage.  As if we add it, we'll want to put it elsewhere.  @nmetulev thoughts?

Reply:
@michael-hawker @nmetulev The button click event need to be registered on the items which are auto-generated in a ListBox which depends on another binding, that appears not flexible to do that by click event, so we decided to use command, or do you have any other cool idea?